removed query cancel from ParseObservable.get()

osamarao opened the pull request:
If this part of the chain is removed then the query starts working else it only responds with ParseException, code 101, OBJECT_NOT_FOUND.

Why is this part of the code necessary? If it is necessary then please guide me as to how to get this working.

Commit:
removed query cancel from ParseObservable.get()

yongjhih commented:
@osamarao , I cannot reproduce your issue. I wrote a sample app: rxparse-app. You can try it with https://github.com/yongjhih/RxParse/blob/c5fa776ff0e64696ac77c6045b50dc35b49e5d94.

You can use your parse application via rxparse-app/build.gradle:

``` gradle
resValue "string", "parse_app_id", "YOUR_PARSE_APP_ID"
resValue "string", "parse_client_key", "YOUR_PARSE_CLIENT_KEY"
```

osamarao commented:
so Andrew, my version isn't much different from yours, however I am not using Lambdas and I am on Java 7. 

Let me share the re-created ParseObservables class, see if you can find any error I made.

https://gist.github.com/osamarao/c3885b83c5fd87246636

yongjhih commented:
@osamarao , I got it, so I force-updated for java7. you can try it.

osamarao commented:
Which class did you force update? I can't see the diff.

yongjhih commented:
Allow rxparse-app can be compiled with java7.

rxparse-app depends on rxparse library that be imported from jcenter 1.0.2.

https://github.com/yongjhih/RxParse/blob/c5fa776ff0e64696ac77c6045b50dc35b49e5d94/rxparse-app/src/main/java/rx/parse/app/MainFragment.java

osamarao commented:
Okay, I'll test again and let you know. 

BTW I also changed the ParseObsevable class and added that as a gist in [this comment](https://github.com/yongjhih/RxParse/pull/2#issuecomment-97351645)
